tools/st2ctl

```diff
@@ -6,7 +6,7 @@ LOGFILE="/tmp/st2_startup.log"
 WEBUI_LOG_FILE="/var/log/st2/st2web.log"
 COMPONENTS="st2actionrunner st2api st2auth st2sensorcontainer st2rulesengine mistral st2web st2resultstracker st2notifier"
 STANCONF="/etc/st2/st2.conf"
-PYTHON=`which python`
```

Comment:
Yeah, I posted on the IRC the other day if it might make sense to ship our own wrapper script which detects the right path to Python 2.7 interpreter and use this as the shebang entry point.

This would be more robust, but I'm not sure if users / system admins would like this approach since it adds some indirection and it's not "unixy".